### ISSUE TITLE
Student Bill—PayPal Gateway Payments Not Posting

### ISSUE THREAD

**Issue by BereaSchool** [parkerj/eduTrac-SIS#107]
In this issue, we tested student #1341.  He made a payment of $5 through the «Pay Now» button in his 20/PR student bill.  The payment was processed correctly through PayPal.  However, on the eduTrac side, the payment is not posting on the student's account even though the message to the student is "Your payment went through successfully and was posted to your account".  See the attached screenshots. Please help.

<img width="1102" alt="Screen Shot 2019-09-23 at 6 51 37 PM" src="https://user-images.githubusercontent.com/35111092/65469347-1ea0d300-de35-11e9-94e1-87ed7eeb8cdf.png">

<img width="1073" alt="Screen Shot 2019-09-23 at 6 53 28 PM" src="https://user-images.githubusercontent.com/35111092/65469384-42fcaf80-de35-11e9-8482-c07dbbbaab6b.png">

**Comment by nomadicjosh** [comment]
There are no error messages that I can see. So, has this been working in the past, and it no longer works?

**Comment by BereaSchool** [comment]
I cannot confirm that it worked in the past.  We are just going by the message the student receives regarding payments being posted on their account.  If the payments do not post, I suggest you change the message to "payment will be reflected on your account in 12-24 hours" or something like that. Thoughts?

**Comment by BereaSchool** [comment]
Any updates on this issue? We still don't have payments posting. Thanks!

**Comment by nomadicjosh** [comment]
Do you have IPN communication enabled in Paypal? If so, is there also a link connected to it?

**Comment by BereaSchool** [comment]
The IPN communication was disabled for some reason.  I edited the settings and enabled it.  I took a screen shot. Is that correct?
<img width="457" alt="Screen Shot 2020-01-28 at 12 42 46 AM" src="https://user-images.githubusercontent.com/35111092/73238502-3aabdc80-4167-11ea-842e-4654293ee93a.png">

**Comment by nomadicjosh** [comment]
Yes, that is correct.

**Comment by BereaSchool** [comment]
Thanks for looking at this.  I believe the reason it was left disabled was due to the message that you have on the PayPal Gateway screen (see screenshot).  I will leave this issue open until we have a PayPal payment made and we can test whether the payment posts into the student's account.  Thanks!
![Screen Shot 2020-01-29 at 10 28 18 AM](https://user-images.githubusercontent.com/35111092/73370441-3e805180-4282-11ea-80cd-cf1ea51f9e5d.png)

**Comment by nomadicjosh** [comment]
Yes, that message should be changed. It is only necessary that IPN is enabled. The notification url isn't necessary because the system will tell Paypal what url to use.
